manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OpenShot.OpenShot
+PackageVersion: 3.0.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- .osp
+ReleaseDate: 2022-12-08
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86.exe
+  InstallerSha256: ED9FEFD35989B512E962DB7207187B3AE4E94CB2DCD4C233443C03BA180C86B6
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
From the validation error, I'm guessing that the x86 version actually needs a 64-bit OS. 
![image](https://user-images.githubusercontent.com/114952565/207116927-112aea6a-f267-415f-adfc-0a717f066719.png)